### PR TITLE
add drone-consider for drone

### PR DESCRIPTION
**Pull request by PierreJeanjacquot**
The drone plugin [drone-tree-config](https://github.com/bitsbeats/drone-tree-config) will be enabled for this repository in the next few days, a consider file `.drone-consider` listing the drone configuration files will be mandatory after the activation.

### CODE REVIEW

**Comment by PierreJeanjacquot** [comment]
NB: merge this to the develop branch after PR merge

**Review by cedric25 — APPROVED** [review]

**Pull request merged**
